vertical-pod-autoscaler/pkg/updater/logic/updater.go

```diff
@@ -111,6 +107,11 @@ func (u *updater) RunOnce() {
 	}
 	timer.ObserveStep("FilterPods")
 
+	if u.evictionAdmission != nil {
+		u.evictionAdmission.LoopInit(controlledPods)
+	}
+	timer.ObserveStep("LoopInit")
```

Comment:
It looks a bit strange now that we have "loop init" step inside the loop.
Since we'd probably want to keep it here maybe we can still rename the metric (AdmissionInit perhaps)?